src/firebase_functions/private/serving.py

```diff
@@ -0,0 +1,114 @@
+"""
+Module used to serve Firebase functions locally and remotely.
+"""
+# pylint: disable=protected-access
+import os
+import inspect
+import enum
+import yaml
+import importlib
+from os import kill, getpid
+from signal import SIGTERM
+
+from flask import Flask
+from flask import Response
+
+from firebase_functions.private import manifest as _manifest
+from firebase_functions.private import util as _util
+
+
+def get_functions():
+    spec = importlib.util.spec_from_file_location("main", "main.py")
+    if spec is not None and spec.loader is not None:
+        module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(module)
+    else:
+        raise Exception(
+            "Firebase Functions for Python could not find the main.py file in your project."
+        )
+    functions = inspect.getmembers(module, inspect.isfunction)
+    firebase_functions = {}
+    for entry in functions:
+        if hasattr(entry[1], "__firebase_endpoint__"):
```

Comment:
Forgive my ignorance. Why is entry being subscripted? Is this worth leaving a comment to explain?